src/core/data/field/initializers/field_user_initializer.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+#define _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
```

Comment:
this constant does not reflects the path so is misleading
it could maybe be simplified `PHARE_FIELD_USER_INITIALIZER_HPP` ?

Reply:
it's typical to include the full path, but obviously convention https://stackoverflow.com/a/4871964/795574

`_PHARE_CORE_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_`